frontend/src/lib/components/ExportButton/exportsLogic.ts

```diff
@@ -31,6 +31,16 @@ export const exportsLogic = kea<exportsLogicType>([
         addFresh: (exportedAsset: ExportedAssetType) => ({ exportedAsset }),
         removeFresh: (exportedAsset: ExportedAssetType) => ({ exportedAsset }),
         createStaticCohort: (name: string, query: AnyDataNode) => ({ query, name }),
+        startReplayExport: (
+            replayId: string,
```

Comment:
still strongly feel that we'll regret introducing `replayId` and it should be `sessionRecordingId`

replay is the product, recording is the thing

Reply:
ah, it's renamed elsewhere... in which case dangling rename here 😊